tests/test-dtls-srtp.conf

```diff
@@ -0,0 +1,192 @@
+# server DTLSv1.2 SRTP default profile
+-u
+-d
+-l ECDHE-RSA-AES256-GCM-SHA384
```

Comment:
@dgarske 
It looks like we have to explicitly set the cipher suite in the test. I just pick these, is there a better alternative? What if the ciphersuite isn't compiled in?

Reply:
Correct, the suites tests uses the cipher suite name to determine if it is compiled in or not and gates on it. I don't see value is choosing more than one. More importantly testing the exported key material for each profile is correct.